data/settings.js

```diff
@@ -2951,6 +2952,22 @@ This supplies the hostname to use in Generic Security Services API
 Use \`"$ALL"\` (with the quotation marks) to allow all keytab entries.`
 	},
 
+	auth_internal_failure_delay: {
+		default: '2secs',
+		values: setting_types.TIME_MSECS,
+		added: {
+			settings_auth_internal_failure_delay_added: false
+		},
+		seealso: [ 'auth_failure_delay' ],
+		text: `
+This is the delay before replying to client when authentication fails with
```

Comment:
This text could actually use a bit of work. "This is the delay" sounds clunky somehow

Reply:
How about just `The delay before ...`